tests/foreman/ui_airgun/test_contentview.py

```diff
@@ -582,12 +582,11 @@ def test_positive_check_composite_cv_addition_list_versions(session):
         })
         assert session.contentview.search(
             composite_cv)[0]['Name'] == composite_cv
-        ccv_values = session.contentview.read(composite_cv)
-        assert len(ccv_values['content_views']['resources']['unassigned']) == 1
-        assert (
-            ccv_values['content_views']['resources']['unassigned'][0]['Version']
-            == 'Always Use Latest (Currently 2.0) 2.0'
-        )
+        ccv_values = session.contentview.read(composite_cv, 'content_views')
+        cv_values = [cv for cv in ccv_values['content_views']['resources']['unassigned']
+                     if cv['Name'] == non_composite_cv]
+        assert len(cv_values) == 1
```

Comment:
How about 
```python
assert non_composite_cv in [cv['Name'] for cv in ccv_values['content_views']['resources']['unassigned']]
```
instead?

Reply:
@abalakh we need to retrieve that row anyway to ensure that only version 2.0 is there (next assert), we do not want to filter that table twice.

Reply:
discussed in irc. anyway, it wasn't blocking comment